interfacesegregationprinciple/HinhTron.java

```diff
@@ -0,0 +1,17 @@
+package solid.interfacesegregationprinciple;
```

Comment:
I think it should be interface.segregation.principle

Reply:
I renamed it to interfacesegregation.principle bc interface is not valid for package name.